components/AppHeader.vue

```diff
@@ -183,3 +185,12 @@ async function logout () {
   }
 }
 </script>
+
+<style scoped>
+.bcgovblue-btn {
+  background-color: #003364;
```

Comment:
Can you use CSS variables (say, in a theme.scss file) and then use them for all the app colours you use?

Reply:
Eg: https://github.com/bcgov/business-filings-ui/blob/main/src/assets/styles/theme.scss